TF_Settings_Web/src/Components/Pages/Camera/QuickSetup/Calibration/CalibrationScreen.tsx

```diff
@@ -124,30 +144,43 @@ const CalibrationScreen: React.FC<CalibrationScreenProps> = ({ isHandPresent, on
     if (location.pathname.endsWith('top')) {
         return (
             <div className="content-container">
-                <CalibrationInstructions progress={progress} containerStyle={{ paddingTop: '5vh' }} />
-                <CalibrationProgressCircle progress={progress} style={{ top: '15.5vh' }} />
-                {displayHandIndicator ? (
-                    <CalibrationHandLostMessage timeToPosSelect={timeToPosSelect} />
-                ) : (
-                    <div style={{ height: handNotFoundHeight }} />
-                )}
-                <CalibrationTutorialVideo videoStyle={{ paddingTop: '3vh' }} />
-                <CalibrationCancelButton onCancel={onCancel} buttonStyle={{ marginTop: '30.5vh' }} />
+                <FullScreenPrompt promptStyle={{ position: 'fixed', top: '5vh' }} />
+                <CalibrationProgressCircle progress={progress} style={{ position: 'fixed', top: '15.5vh' }} />
+                <img
```

Comment:
Had this thought before but didn't get round to it - could move the touch circle image inside CalibrationProgressCircle?

Reply:
Done